README.md

```
@@ -644,11 +644,7 @@ Image processing with OpenCV works on the client side.
 
 ### Streamlit (Snowflake)
 
-[<img src="https://streamlit.io/images/brand/streamlit-mark-color.png" height="100" />](https://streamlit.io/) [<img src="https://docs.snowflake.com/_images/logo-snowflake-sans-text.png" height="100" />](https://www.snowflake.com/)
-
-### TestMu AI
-
-[<picture><source media="(prefers-color-scheme: dark)" srcset="https://assets.testmuai.com/resources/images/logos/white-logo.png"><source media="(prefers-color-scheme: light)" srcset="https://assets.testmuai.com/resources/images/logos/black-logo.png"><img alt="TestMu AI logo" src="https://assets.testmuai.com/resources/images/logos/white-logo.png" height="85"></picture>](https://www.testmuai.com/?utm_medium=sponsor&utm_source=stlite)
+[<img src="https://streamlit.io/images/brand/streamlit-mark-color.png" height="100" />](https://streamlit.io/) [<img alt="Snowflake" src="https://docs.snowflake.com/images/favicon/apple-touch-icon.png" height="100" />](https://www.snowflake.com/)
```

markdownlint-cli2 (0.23.2)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)